src/index.html

```diff
@@ -11,7 +11,198 @@
+        <div class="form-field">
+          <label for="email">
+            E-mail:
+            <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            multiple
+            required>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="pass">
+            Password:
+            <input
+            type="password"
+            name="password"
+            id="pass"
+            minlength="5"
+            maxlength="32"
+            required>
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-set">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label for="pass">
+            <input
+              type="radio"
+              name="cats"
+              value="y">
+            Yes
+          </label>
+          <label for="pass">
+            <input
+              type="radio"
+              name="cats"
+              value="n">
+            No
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="color">
+            What is your favorite color?
+            <input
+            type="color"
+            name="color"
+            id="color">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="time">
+            What time do you go to bed?
+            <input
+            type="time"
+            name="time"
+            id="time">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="cars">
+            What are your favorite brand of cars?
+            <select
+              name="cars"
+              id="cars"
+              size="4">
+              <option value="bmw" selected>BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+              <option value=""></option>
+            </select>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="rate">
+            How do you rate our work?
+            <input
+            type="range"
+            name="rate"
+            id="rate">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-set">
+        <legend>Additional info</legend>
+        <div class="form-field">
+          <label for="text">
+            Comments:
+            <textarea
+              name="text"
+              id="text"
+              cols="25"
+              rows="2">
+            </textarea>
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="how">
+            Would you recommend us?
+            <select
+              name="opinion"
+              id="how">
+              <option value="y">yes</option>
+              <option value="n">no</option>
```

Comment:
make value understandable

```suggestion
              <option value="yes">yes</option>
              <option value="no">no</option>
```